cms/tests/test_management.py

```diff
@@ -50,7 +50,7 @@ def test_list_apphooks(self):
         command = cms.Command()
         command.stdout = out
         command.handle("list", "apphooks", interactive=False)
-        self.assertEqual(out.getvalue(), "SampleApp\n")
+        self.assertEqual(out.getvalue(), "SampleApp (draft)\n")
```

Comment:
single/double quotes are still a bit messy in the CMS. I prefer single to double but the module is almost all double and I think it's better to do a global cleanup in 3.3